webplugin/js/app/zendesk-chat-service.js

```diff
@@ -85,4 +103,33 @@ function ZendeskChatService() {
             },
         });
     };
+
+    _this.handleZendeskAgentFileSendEvent= function (event) {
+        window.console.log("handleZendeskAgentFileSendEvent ",event);
+
+        var messagePxy = {
+            fileAttachment: event.attachment,
+            fromUserName: event.nick.split(":")[1],
+            groupId: CURRENT_GROUP_DATA.tabId,
+            auth:window.Applozic.ALApiService.AUTH_TOKEN
+        };
+
+        return mckUtils.ajax({
+                url: Kommunicate.getBaseUrl() + "/rest/ws/zendesk/file/send",
+                type: 'post',
+                contentType: 'application/json',
+                data: JSON.stringify(messagePxy),
+                headers: {
+                    'x-authorization': window.Applozic.ALApiService.AUTH_TOKEN,
+                },
+                success: function (result) {
+                    console.log("Sent File message data to the server ", result);
+                    typeof callback == 'function' && callback(agentUserName);
+                },
+                error: function (err) {
+                    console.log('err while sending File message data to the server');
```

Comment:
some places it is window.console.log and some places it is console.log
which one is correct? and what's the purpose of using 2 different ways?

Reply:
Both are technically the same thing. As 'window' is global object for all browsers. 

Ok, Then I am using one format in all places (in this file).